__tests__/app.test.js

```diff
@@ -163,3 +164,43 @@ describe('4. GET - /api/users', () => {
       });
   });
 });
+
+describe('5. GET - /api/articles', () => {
+  test('Status: 200 // Returns array of article objects', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body;
+        expect(articles).toBeInstanceOf(Array);
+      });
+  });
+  test('Status: 200 // Article objects contain props for author, title, topic, article_id, created_at, votes, and comment_count', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body;
+        articles.forEach((article) =>
```

Comment:
If you're using a forEach in a test you should be asserting that the length of the array is at least bigger than 0

Reply:
Added!